docs/plugins/lamp-plugin.md

```diff
@@ -27,6 +27,23 @@ The Probo [Drupal plugin](/plugins/drupal-plugin/), [WordPress plugin](/plugins/
 | `phpConstants` | A hash of constants, such as {const1: 'const1Value', const2: 'const2Value',}. This will overwrite any other auto_prepend_file directives in your php.ini. Accepts a **hash** value. |
 | `phpMods` | An array of php5 modules to enable (should be installed via installPackages if needed). Accepts an **array** value. |
 
+### Modify PHP Environment Options in a Build
+Probo builds have their own isolated `php.ini` files. Specific PHP options for your build can be modified using the `phpIniOptions` configuration option in  your `.probo.yaml` file. The `phpIniOptions` configuration option must be paired with a LAMPApp compatible [Probo Plugin](https://docs.probo.ci/plugins/) to set specific PHP settings within the plugin's steps. 
+
+See below for an example YAML config that sets custom `upload_max_filesize`, `post_max_size`, and `memory_limit` PHP settings values for a Probo Build using the [Probo LAMPApp plugin](/plugins/lamp-plugin/).
+
+{% highlight yaml %}
```

Comment:
~Can you instead add this as a recipe and embed that recipe? See the bottom of https://github.com/ProboCI/probo-docs/blob/master/docs/plugins/drupal-plugin.md for an example of how that works.~

~Adding as a recipe means this will show up in the recipes list, too (https://docs.probo.ci/recipes/).~

We actually have an example of this at the bottom of the page. Since this page is getting a little complex, maybe we should move those examples below the related headers. E.g., "Setting LAMPApp PHP Configuration Options on a Drupal Installation" is moved below "PHP Options", we make an example for "Apache Options", etc.

Reply:
The problem is that the example we have had up is wrong. The options need to be inside the LAMP plugin. I'll update that recipe instead and just note the example in the copy above.